packages/vscode-extension/PRERELEASE.md

```diff
@@ -2,6 +2,48 @@
 
 ## Changelog
 
+### September 26, 2023
+
+#### New Features
+
+- **Sample App Gallery Search**: We have introduced the ability to search for sample apps within the Sample App Gallery.
+    ![Sample Gallery](https://github.com/OfficeDev/TeamsFx/assets/11220663/0b5636da-aa21-4624-82c1-2824f2557e9f)
+
+- **Custom Search Results Template Optimization**: We've optimized the `Custom Search Results` project template to seamlessly integrate with Microsoft 365 Copilot. To utilize this feature, please enable the `Develop Copilot Plugin` feature setting via Visual Studio Code in the [User and Workspace Settings](https://code.visualstudio.com/docs/getstarted/settings) and enroll in the Microsoft 365 Copilot [Early Access Program](https://aka.ms/PluginsEarlyAccess).
+    ![Copilot Plugin](https://github.com/OfficeDev/TeamsFx/assets/11220663/67c22709-4c70-4d17-b9a3-9e1629ceded1)
+
+#### New Additions to the Sample App Gallery
+
+- **SSO Enabled Tab via APIM Proxy**: This addition showcases how to build a single-page web application with a single sign-on feature leveraging Azure API Management service as a proxy to perform on-behalf-of flow, eliminating the need for a dedicated backend.
+    ![APIM-Architecture](https://github.com/OfficeDev/TeamsFx-Samples/assets/63089166/a256f1ab-1b23-4264-9f0d-ed8ff45aea09)
+
+- **Large Scale Notification Bot**: This addition demonstrates the architecture of a Teams notification bot app created by Teams Toolkit. It enables sending individual chat messages to a large number of users in a tenant and relies on Azure services such as [Durable Function](https://learn.microsoft.com/azure/azure-functions/durable/durable-functions-overview?tabs=csharp-inproc) and [Service Bus Queue](https://learn.microsoft.com/azure/service-bus-messaging/service-bus-queues-topics-subscriptions#queues) to handle high-volume and high-speed notification messaging.
```

Comment:
This sample is still going through onboarding process and expected to be released in Oct. through sample release process.

Reply:
OK, I'll remove it. One more I want to mention here that we are now decoupling sample & TTK releases, but I still recommend we sync the sample onboarding timeline along with the TTK release or prerelease cycle, so that we have a chance to announce new sample additions in release note and blogs.

Reply:
Sure, the sample release timeline is aligned with TTK, the decoupled design provides more flexibility.